modulefiles/hera.intel-run.lua

```diff
@@ -6,6 +6,7 @@ prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-s
 local stack_intel_ver=os.getenv("stack_intel_ver") or "2021.5.0"
 local stack_impi_ver=os.getenv("stack_impi_ver") or "2021.5.1"
 local grads_ver=os.getenv("grads_ver") or "2.2.1"
+local perl_ver=os.getenv("grads_ver") or "5.38.0"
```

Comment:
I think you want "perl_ver" not "grads_ver".

Reply:
Good catch, thanks!